setup.py

```diff
@@ -26,7 +26,7 @@
     author_email='data-analytics-team@palmetto.com',
     url='https://github.com/palmetto/palm-dbt',
     packages=find_namespace_packages(include=['palm', 'palm.*']),
-    package_data={'': ['*.md', '*.sql', '*.yaml', '*.yml']},
+    package_data={'': ['palm/plugins/dbt/**/*']},
     install_requires=[
 		'palmcli>=2.0.0'
```

Comment:
This will also need to be bumped to v2.1.0. We should do this by reading the requirements.txt in here - see `palm-cli` setup.py for an example of how to do that